node/node.go

```diff
@@ -180,7 +190,11 @@ func (n *Node) Start(ctx context.Context) error {
 			return fmt.Errorf("failed to register the operator: %w", err)
 		}
 	} else {
-		n.Logger.Info("The node has successfully started. Note it's not opt-in to EigenDA yet (it's not receiving or validating data in EigenDA). To register, please follow the EigenDA operator guide section in docs.eigenlayer.xyz")
+		eigenDAUrl := "unsupported chain"
+		if n.ChainID.String() == goerliChainID {
+			eigenDAUrl = "https://goerli.eigenlayer.xyz/avs/eigenda"
```

Comment:
This can be a constant map from chain id to URL defined at the top

Reply:
oh that's a better idea. and log message can be manipulated if chain not found

Reply:
done